### DISCUSSION TITLE
Support for React-Native Windows?

### DISCUSSION THREAD

**Discussion by LyDawei** [10play/10tap-editor#214] [General]
Do we have any traction to have support for React-Native Windows? I have a need for an RTF editor, unfortunately a huge chunk of my user base uses Windows devices and we have a need for it to work in React-Native Windows. I've looked into react-native-pell-rich-editor which allows me to have an editor but toolbar is broken so I'm exploring my options. I really like 10tap due to it being based on tiptap which I've used in the past for other projects.

Thanks in advance.

**Comment by 17Amir17** [comment]
Hey!
Currently there is no windows support, but we do have react-native-web support so maybe you can hack something together there

**Reply by 17Amir17** [reply]
Hey @LyDawei 
Due to multiple requests from our enterprise client we added support. We're considering free but for now this is available on our enterprise plan. If you are interested feel free to contact us at email@10play.dev
